### DISCUSSION TITLE
Can't show "Rencent"

### DISCUSSION THREAD

**Discussion by Okoonzz** [nunocoracao/blowfish#784] [Q&A]
I don't understand why, when I publish "Posts," my homepage doesn't show "Recent" even though I have enabled it in the menu and homepage settings.

![image](https://github.com/nunocoracao/blowfish/assets/90641599/b4ad967d-e98b-42ba-91aa-85753e1bff81)

![image](https://github.com/nunocoracao/blowfish/assets/90641599/4cfb0d62-d37c-491a-a8ef-5e9c5ea53e19)

![image](https://github.com/nunocoracao/blowfish/assets/90641599/ae9f5aa2-f9bb-485b-987d-9467b30b3fe1)

![image](https://github.com/nunocoracao/blowfish/assets/90641599/140e05bd-9bb6-4d8e-8d16-fd9bcc854b89)

![image](https://github.com/nunocoracao/blowfish/assets/90641599/070070c3-62ca-40e2-a840-c795e28726e9)

**Comment by nunocoracao** [comment]
Hello @Okoonzz does each article have a date field?
